pkg/postage/batchstore/store.go

```diff
@@ -288,6 +305,80 @@ func (s *store) Reset() error {
 	return nil
 }
 
+// migrate converts batchstore to the refactored version, removes unused indexes, computes radius, and
+// and calculates available capacity.
+func (s *store) migrate() error {
```

Comment:
not sure why this is needed.... i would recommend to just delete all batchstore keys as part of a statestore migration and force the batchstore to resync all chain data.

Reply:
fair enough, my thought process was that we could get away with not having to force users to a full resync, but it is the simplest and the least risky method.